tests/test_basics.py

```diff
@@ -566,7 +566,7 @@ def _fake_import(name, *args):
     def _fake_is_library_loaded(name, *args):
         return False
 
-    monkeypatch.delenv("PYTEST_QT_API", raising=False)
```

Comment:
Note that we need to keep supporting `PYTEST_QT_API` for backward compatibility, so we should not change these tests.